### DISCUSSION TITLE
Issue regarding Github Student Pack

### DISCUSSION THREAD

**Discussion by shivamojha10** [community/community#77170] [GitHub Education]
### Student devoloper pack,

![image](https://github.com/community/community/assets/112488813/7781de83-170d-4026-85a7-7892acaefa4d)
Hi everyone! It's been more than 72 hours, and I still haven't got my Github student package. I've read posts about similar issue here on community forum, and since this isn't a busy time, and it's been 3 days already, I am confused why I still haven't got my package?

Please help!

**Comment by CheetahCodes21** [comment]
I understand your concern about the delay in receiving your GitHub Student Developer Pack. While the verification process typically takes up to 72 hours, there could be occasional delays due to various factors, such as a high volume of applications or technical issues.

Here are some steps you can take to resolve the issue:

1. **Check your application status:** Visit the GitHub Education Pack application page and check your application status. If it says "Approved," your benefits should be activated within 24 hours.

2. **Contact GitHub Support:** If your application is still pending or you haven't received your benefits after 24 hours of approval, reach out to GitHub Support for further assistance. They can investigate your case and provide an update on the status of your application.

Here's how to contact GitHub Support:
- Go to the GitHub Help page.
- In the search bar, type "Student Developer Pack" and press Enter.
- Select the relevant article or discussion thread.
- If there's no relevant article or discussion thread, click the "Contact Support" button.
- Follow the prompts to submit a support ticket.

Provide them with your application details and explain the delay you've experienced. They'll be able to review your case and provide further assistance.
